components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/context/AuthenticationContext.java

```diff
@@ -103,6 +105,10 @@
      */
     private AuthenticatorStateInfo stateInfo;
 
+    private String userTenantDomainHint;
+
+    private String loginTenantDomain;
```

Comment:
Which tenant domain would this be?
loginTenantDomain feels a bit ambigious

Reply:
It's explained in the getter below https://github.com/wso2/carbon-identity-framework/pull/3338/files#diff-df1cf733997e1697bde833bbe6aad3866c7598433619a0ee1581be11f51927f7R690-R696